pybullet_rendering/render/panda3d.py

```diff
@@ -100,14 +101,17 @@ def update_scene(self, scene_graph, materials_only):
                 if not filename:
                     continue
                 shape = self._loader.load_model(filename)
-                shape.set_mat(Mat4(*pb_shape.pose.matrix))
-                shape.reparent_to(node)
                 if shape.has_material:
                     shape.set_material(PbMaterial(pb_shape.material), 1)
                     texture_id = pb_shape.material.diffuse_texture
                     if texture_id > -1:
                         texture = scene_graph.texture(texture_id)
                         shape.set_texture(texture.filename)
+                pose = pb_shape.pose
+                shape.setPos(*pose.origin)
+                shape.setQuat(Quat(*pose.quat))
+                shape.setScale(*pose.scale)
```

Comment:
This change was required, because using set_mat for the Capsule type did not work properly